app/src/main/java/io/intrepid/contest/base/BaseFragmentActivity.java

```diff
@@ -26,7 +26,7 @@ protected void onCreate(Bundle savedInstanceState) {
             Fragment fragment = createFragment(intent);
             getSupportFragmentManager()
                     .beginTransaction()
-                    .add(R.id.fragment_container, fragment)
+                    .replace(R.id.fragment_container, fragment)
```

Comment:
I was thinking it might be possible that the fragment was already added and then we would have added it multiple times. Perhaps that was a fallacy.